src/reactable/table.jsx

```diff
@@ -133,6 +133,36 @@ export class Table extends React.Component {
         this.tfoot = tfoot;
 
         this.initializeSorts(props);
+        this.initializeFilters(props);
+    }
+
+    initializeFilters() {
+        this._filterable = {};
+        // Transform filterable properties into a more friendly list
+        for (let i in this.props.filterable) {
```

Comment:
okay so actually, using for...of isn't gonna work here thanks to some phantom phuckery (shoutout to @jkap for that) so at least for the scope of this PR we're gonna have to stick with this